multimodal/setup.py

```diff
@@ -33,13 +33,13 @@
     "evaluate>=0.2.2,<0.4.0",
     "accelerate>=0.9,<0.17",
     "timm>=0.6.12,<0.7.0",
-    "torch>=1.9,<1.14",
-    "torchvision<0.15.0",
+    "torch>=1.11,<2.1",
```

Comment:
This change makes user install torch 2.0 by default. Is the CI still testing torch 1.13.1? If so, there is some inconsistency between what we test and what users use.

Reply:
does this thread answer your question? https://github.com/autogluon/autogluon/pull/3190#discussion_r1183987923

after this merges I think there are no explicit tests that run with pytorch 1.x anymore